ci/integration-tests-mac.groovy

```diff
@@ -40,25 +40,12 @@ pipeline {
 
         stage("Launch AWS Cluster") {
             steps {
-                withCredentials(credentials) {
-                    script {
-                        master_ip = sh(script: '''docker run --rm -v $PWD:/usr/src -w /usr/src \
-                                      -v ${CLI_TEST_SSH_KEY_PATH}:${CLI_TEST_SSH_KEY_PATH} \
-                                      -e DCOS_TEST_INSTALLER_URL \
-                                      -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY \
-                                      -e DCOS_USERNAME -e DCOS_PASSWORD \
-                                      -e DCOS_TEST_LICENSE -e CLI_TEST_SSH_KEY_PATH \
-                                      python:3.7-stretch bash -c " \
-                                        cd scripts; \
-                                        python3 -m venv env; \
-                                        source env/bin/activate; \
-                                        pip -q install --upgrade pip==18.1 setuptools; \
-                                        pip -q install -r requirements.txt; \
-                                        ./launch_aws_cluster.py create"''',
-                                returnStdout: true).trim()
-                    }
-                    stash includes: 'scripts/config.json', name: 'aws-config'
-                }
+//                withCredentials(credentials) {
```

Comment:
What's the story here? Is it supposed to be commented out?

Reply:
From description:
> disables E2E tests for Mac and Windows. It will be fixed in separated PR

For some reason we get connect timed out for Mac and Windows, for linux too when using py37 docker image. Right now I have no clue why. Can't reproduce it locally :(